sdk/core/azure-core/src/main/java/com/azure/core/util/Base64Url.java

```diff
@@ -40,7 +40,13 @@ public Base64Url(byte[] bytes) {
 
     private static byte[] unquote(byte[] bytes) {
         if (bytes != null && bytes.length > 1) {
-            bytes = unquote(new String(bytes, StandardCharsets.UTF_8)).getBytes(StandardCharsets.UTF_8);
```

Comment:
Do we need the `unquote(String)` overload now?

Reply:
Yes as there are other APIs that call into the String-based method